example/eblob_backend.cpp

```diff
@@ -66,14 +66,30 @@ int blob_check_corrupted_stamp(void *buffer, size_t buffer_size) {
 		       dc.position == 0;
 	};
 
+	if (!check_dc_header(*dc)) {
+		// eblob_disk_control signature check failed
```

Comment:
I would rephrase this and comments below to: `eblob_disk_control signature wasn't found the key looks normal`